### PR TITLE
Skip release archive

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind deprecation

#### What this PR does / why we need it:
Don't archive the release any more and move the announcement to the release artifacts (`kubernetes-release/release/v1.x.y`).

#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/release/issues/3716
#### Special notes for your reviewer:
Test run: https://console.cloud.google.com/cloud-build/builds/30395f84-a680-4315-a144-7049da5f7805

Output: https://console.cloud.google.com/storage/browser/kubernetes-release-gcb/release/v1.31.1
#### Does this PR introduce a user-facing change?

```release-note
Skip release archive for `krel release`. `krel announce send` will now use the announcement from the release artifacts rather than the archive.
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
I don't think we need to write the log files, I can follow-up on that.

PTAL @kubernetes/release-engineering

File: cmd/krel/cmd/announce_send.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func runAnnounce(opts *sendAnnounceOptions, announceRootOpts *announceOptions, r
 
 	tag := util.AddTagPrefix(announceRootOpts.tag)
 	u := fmt.Sprintf(
-		"%s/archive/anago-%s/announcement.html",
+		"%s/release/%s/announcement.html",
```
Comment (ameukam) [inline cmd/krel/cmd/announce_send.go:120]: Switching this is not impactful for the patch releases ?

**Comment by ameukam** [comment]
/lgtm
/hold

In case other releng folx want to chime in.

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Comment by saschagrunert** [comment]
@kubernetes/release-engineering PTAL

**Pull request merged**